server/api/routes.md

```diff
@@ -0,0 +1,49 @@
+**USER: (/api/user)**
+
+* register guest user (POST) - occurs as soon as any user visits site (any page). Sets cookie (sessionId)
+* create user account (POST)
+* authenticate registered user (POST) - sets cookie (some version of userId)
+* retrieve single user, or all users (GET)
+* update user (PUT)
+* delete user account (DELETE)
+
+**CART/CART-ITEM: (/api/cart)**
+* create new cart (POST) - use userId (new cart should be created as soon as new user enters app)
+* retrieve single cart, or all carts (GET)
+* add to cart (POST) - creates new CartItem, uses productId of selected product, cartId
+* retrieve single CartItem, or all CartItem(s) (GET)
+* edit cart item (PUT)
+* delete cart item (DELETE)
+* clear cart (PUT)
+
+**PRODUCT: (/api/products)**
+* add new product (POST) - this should create new price in the process
```

Comment:
The thunk will create the price, right?

Reply:
Dismissing this since we no longer have Price model